src/main/java/com/reckue/post/services/realizations/PollNodeServiceRealization.java

```diff
@@ -0,0 +1,190 @@
+package com.reckue.post.services.realizations;
+
+import com.reckue.post.exceptions.ModelAlreadyExistsException;
+import com.reckue.post.exceptions.ModelNotFoundException;
+import com.reckue.post.models.PollNode;
+import com.reckue.post.repositories.PollNodeRepository;
+import com.reckue.post.services.PollNodeService;
+import com.reckue.post.utils.Generator;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Class PollNodeServiceRealization represents realization of PollNodeService.
+ *
+ * @author Grigoriev Viktor
+ */
+@Service
+@RequiredArgsConstructor
+public class PollNodeServiceRealization implements PollNodeService {
+
+    private final PollNodeRepository pollNodeRepository;
+
+    /**
+     * This method is used to create an object of class PollNode.
+     * Throws {@link ModelAlreadyExistsException} in case if such object already exists.
+     *
+     * @param node object of class PollNode
+     * @return node object of class PollNode
+     */
+    @Override
+    public PollNode create(PollNode node) {
+        node.setId(Generator.id());
+        if (!pollNodeRepository.existsById(node.getId())) {
+            return pollNodeRepository.save(node);
+        } else {
+            throw new ModelAlreadyExistsException("PollNode already exists");
+        }
+    }
+
+    /**
+     * This method is used to update data in an object of class PollNode.
+     * Throws {@link ModelNotFoundException} in case
+     * if such object isn't contained in database.
+     * Throws {@link IllegalArgumentException} in case
+     * if such parameter is null.
+     *
+     * @param node object of class PollNode
+     * @return node object of class PollNode
+     */
+    @Override
+    public PollNode update(PollNode node) {
+        if (node.getId() == null) {
+            throw new IllegalArgumentException("The parameter is null");
+        }
+        if (!pollNodeRepository.existsById(node.getId())) {
+            throw new ModelNotFoundException("PollNode by id " + node.getId() + " is not found");
+        }
+        PollNode savedPollNode = PollNode.builder()
+                .id(node.getId())
+                .title(node.getTitle())
+                .build();
```

Comment:
Considering that there are **items** in the node, they should be here into builder too.